packages/go/index.ts

```diff
@@ -536,7 +550,16 @@ Learn more: https://vercel.com/docs/runtimes#official-runtimes/go`
   const analyzed: Analyzed = JSON.parse(analyzedRaw);
 
   await Promise.all([
-    copyEntrypoint(entrypointWithExt, tmpPackage),
+    copyEntrypoint(
+      [
+        relative(
+          goModAbsPathDir,
+          (files[entrypointWithExt] as FileFsRef).fsPath
```

Comment:
Same here, instead of using `files` use `join(workPath, entrypointWithExt)`

Reply:
Ok, I tried it and it works fine, I will use `path.join` instead of `files`.